### PR TITLE
[15.0][MIG] contract_price_revision: Migration to version 15.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v15
- Add ir.model.access.csv
- Fix error comparing dates

@Tecnativa

TT45292

@pedrobaeza @carlosdauden

### CODE REVIEW

File: contract_price_revision/models/contract_line.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, fields, models
+from odoo.fields import Date
```
Comment (pedrobaeza) [inline contract_price_revision/models/contract_line.py]: No need of importing a special variable, use `fields.Date.today()` instead.

File: contract_price_revision/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_contract_price_revision_wizard,access_contract_price_revision_wizard,model_contract_price_revision_wizard,base.group_user,1,1,1,1
```
Comment (pedrobaeza) [inline contract_price_revision/security/ir.model.access.csv]: You should mimic here the permissions of the contract model, not leaving this open for any internal user.

File: contract_price_revision/wizards/contract_price_revision.py [hunk 3]
```diff
@@ -69,8 +69,8 @@ def action_apply(self):
             line.stop(date_end)
             new_line = line.copy(self._get_new_line_value(line))
             line.update({"successor_contract_line_id": new_line.id})
-        action = self.env["ir.actions.act_window"].for_xml_id(
-            "contract", "action_customer_contract"
+        action = self.env["ir.actions.actions"]._for_xml_id(
```
Comment (pedrobaeza) [inline contract_price_revision/wizards/contract_price_revision.py]: Don't replace `ir.actions.act_window`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by carolinafernandez-tecnativa** [comment]
ping @pedrobaeza

**Comment by carolinafernandez-tecnativa** [comment]
ping @carlosdauden

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**